django/db/backends/oracle/introspection.py

```diff
@@ -22,15 +23,11 @@ class DatabaseIntrospection(BaseDatabaseIntrospection):
         cx_Oracle.TIMESTAMP: 'DateTimeField',
     }
 
-    try:
+    with suppress(AttributeError):
         data_types_reverse[cx_Oracle.NATIVE_FLOAT] = 'FloatField'
```

Comment:
I guess the `AttributeError` is for `cx_Oracle.NATIVE_FLOAT` -- this is probaby obsolete now that cx Oracle 5.2+ is supported.